scripts/openmc-make-depletion-chain

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+from zipfile import ZipFile
+
+from openmc._utils import download
+import openmc.deplete
+
+
+URLS = [
```

Comment:
This website supports downloading these resources over HTTPS.

Reply:
Fixed in #1424